new dir to encapsulate upload command

Scott-Canning opened the pull request:
Created `src/commands` directory to encapsulate commands outside of `cli.js` and moved the `upload` command

aboutlo commented on src/cli.js:
```diff
@@ -1,37 +1,11 @@
-    },
-  )
+  .command(upload.command, upload.description, upload.builder, upload.handler)
   .option('private-key', {
```
Please try to move both the options in the `builder` function in the upload command.

aboutlo commented on src/commands/upload.js:
Let's rename it as `uploadCommand.js` given we already have the `upload.js` file with the actual implementation

aboutlo reviewed (changes requested):
Looks good a couple of potential improvement before merging it

Scott-Canning commented:
Changed `commands/upload.js` to `commands/uploadCommand.js` and moved `upload` options into the associated `builder` function.

Test locally with `npx . upload build/ --private-key ${PRIVATE_KEY} --dry-run`

aboutlo reviewed (approved):
LGTM

AronVanAmmers commented:
:tada: This PR is included in version 0.1.4 :tada:

The release is available on:
- [GitHub release](https://github.com/OutlierVentures/arweave-bundler/releases/tag/v0.1.4)
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/@outlierventures/arweave-bundler/v/0.1.4)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
